src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -1956,6 +1960,26 @@ public override object VisitField(FieldSymbol symbol, TypeCompilationState argum
                 return null;
             }
         }
+
+#if DEBUG
+        private static bool IsEmptyRewritePossible(BoundNode node)
```

Comment:
Consider just marking this method/class with `Conditional` and removing the `#if`s

Reply:
nit: FWIW, switching to a conditional `void` method "AssertEmptyRewritePossible" would have solved that